pkg/controller/operators/olm/operator.go

```diff
@@ -396,8 +420,8 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 	op.lister.CoreV1().RegisterNamespaceLister(namespaceInformer.Lister())
 	namespaceInformer.Informer().AddEventHandler(
 		&cache.ResourceEventHandlerFuncs{
-			DeleteFunc: op.namespaceAddedOrRemoved,
-			AddFunc:    op.namespaceAddedOrRemoved,
+			DeleteFunc: op.namespacesAddedOrRemoved,
```

Comment:
Do we need this to happen in parallel to `op.syncNamespace`?

Reply:
That's a great idea.